ansible/eos_install_config

```diff
@@ -107,7 +124,8 @@ def main():
             password=dict(required=True),
             use_ssl=dict(required=False, default=True),
             config_file=dict(required=True),
-            commit_changes=dict(required=True),
+            commit_changes=dict(required=False, default=False),
+            replace_config=dict(required=False, default=False),
```

Comment:
No, I didnt notice... And I kind of prefer to have it as before. Mainly to make sure you specify what you want to do. If you forget it, you should get an error. Not a default option. I think that parameter kind of important to make sure it´s not forgotten ; )